features/users/hide_from_search_engines.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Hiding users and their content from search engines
+  Scenario: Hidden users' user pages are disallowed for search engine indexing
+    Given I have no users
```

Comment:
Just wondering -- why do we need to start with no users?

Reply:
A misunderstanding on my part, it wasn't actually needed.